### PR TITLE
Remove merge queue from query is feature is disabled

### LINKED ISSUE #356: Latest version incompatible with older GitHub Enterprise Server usages.
#347 added Merge Queue support for this tool. Unfortunately, it seems that this change is incompatible with Github Enterprise Servers that don't have this feature enable (I believe it was just released this past July). When running the tool, we get the following panic:

```
panic: query PullRequests.viewer.pullRequests.nodes.mergeQueueEntry: Field 'mergeQueueEntry' doesn't exist on type 'PullRequest'

goroutine 1 [running]:
github.com/ejoffe/spr/github/githubclient.check({0x104f1fca8, 0x1400000e6d8})
       .../spr/github/githubclient/client.go:671 +0xe8
github.com/ejoffe/spr/github/githubclient.(*client).GetInfo(0x1400019c9a8, {0x104f23410?, 0x105239160?}, {0x104f23330, 0x1400019c978})
        .../spr/github/githubclient/client.go:183 +0xc0
github.com/ejoffe/spr/spr.(*stackediff).fetchAndGetGitHubInfo(0x140001a4de0, {0x104f23410, 0x105239160})
       .../spr/spr/spr.go:490 +0x14c
github.com/ejoffe/spr/spr.(*stackediff).UpdatePullRequests(0x140001a4de0, {0x104f23410?, 0x105239160}, {0x0, 0x0, 0x0}, 0x0)
       .../spr/spr/spr.go:137 +0x70
...
```

Is there a way to make this query optional such that the tool remains compatible with older Github Enterprise instances? I'm happy to submit a PR to do so, but am not too familiar with how to accomplish this in GraphQL.

### PR DESCRIPTION
**Pull request by ejoffe**
fixes #356

commit-id:0219e1da

### CODE REVIEW

**Review by pleicht — COMMENTED** [review]
Thanks a bunch for making this change!  I think the condition for when to add the MergeQueue to the operation string needs to be inverted

File: github/githubclient/gen/genclient/operations.go [hunk 1]
```diff
@@ -91,7 +92,7 @@ func (c *gqlclient) PullRequests(ctx context.Context,
 				repository {
 					id
 				}
-				mergeQueueEntry {
+				mergeQueueEntry @skip(if: $use_merge_queue) {
```
[the remark this answers is not in the document]
Comment (pleicht) [reply github/githubclient/gen/genclient/operations.go:95]: This file is probably generated from the above.  Still completely new to Go

**Comment by pleicht** [comment]
> Looks right to me. The condition is to skip this field if use_merge_queue == true.

I tested it locally when building this branch and I get the error for merge queue.  I think we only want to generate the MergeQueue section if `use_merge_queue` is true, if false we exclude

**Comment by ejoffe** [comment]
Should be correct now. Good catch.

**Comment by ejoffe** [comment]
I guess the skip directive doesn't work in this case, will try another approach.

**Comment by pleicht** [comment]
> I guess the skip directive doesn't work in this case, will try another approach.

👍, thanks for the help and spr in general, I really enjoy using it!

**Review by pleicht — APPROVED** [review]
Tested and confirmed it is working for me. Thanks!

**Pull request merged**